pipelines/README.md

```diff
@@ -93,6 +93,6 @@ oc create -f test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-tensorflow
 > If using one of the provided `PipelineRun` files, make sure all of the parameters match your Git environment.
 
 ```bash
-oc apply -k gitops-update-pipeline/
-oc create -f gitops-update-pipeline/gitops-update-pipelinerun-tensorflow-housing.yaml
+oc apply -k tekton/gitops-update-pipeline/
+oc create -f tekton/gitops-update-pipeline/gitops-update-pipelinerun-tensorflow-housing.yaml
```

Comment:
I'd personally prefer all of them start with `tekton/` over that and keep the README where it is since the user is not likely to use the other subdirectories often.

I think we need a README in `pipelines/` to tell the user what it's for, and pointing to _another_ README in `pipelines/tekton` feels a little annoying to me e.g. You read the README at the root of the repo, which points you to a README in `pipelines/`, which points you to a README in `pipelines/tekton`.

idk tho ¯\\_(ツ)_/¯

Reply:
Updated, thanks.